cli_tools/daisy/main.go

```diff
@@ -282,3 +285,33 @@ func main() {
 		}
 	}
 }
+
+func printPerfProfile(workflow *daisy.Workflow) {
+	if !*printPerf {
+		return
+	}
+
+	timeRecords := workflow.GetStepTimeRecords()
+	if len(timeRecords) == 0 {
+		return
+	}
+
+	wfStartTime := time.Now()
+	wfEndTime, _ := time.Parse(time.RFC3339, "0001-01-01T00:00:00+00:00")
```

Comment:
Whats the point of this? Why not just 'var wfEndTime time.Time' ?

Reply:
Since I didn't know that is the default value of time... changed to time.Time{} accordingly